docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,22 +1,157 @@
-# Cody Ignore
+# Cody Context Filters
 
-<Callout type="note">This feature is currently in the Experimental stage and only supported with VS Code extension.</Callout>
+<p className="subtitle">This documentation helps you control what context from your codebase is used by Cody. You can do it via </p>
 
-This guide will walk you through the process of ignoring specific files or folders in your codebase when using Cody. 
+- Cody Context Filters (Cody Enterprise Only)
+- Cody Ignore File (All Cody users)
```

Comment:
@MaedahBatool Cody Ignore File is still in Experimental. Can we capture that up front somewhere?